README.md

```diff
@@ -763,8 +786,10 @@ The signature requester, prover in this case, creates a blind signature request.
 randomness `blinding` that goes into the commitment. This randomness is later used
```

Comment:
This is a place where BBS and BBS+ differ. With BBS, there is no `blinding` (the commitment is computationally hiding, though it can be made perfectly hiding by adding a dummy attribute). Can you please mention it here. And see comment below on `generateRequest`